### DISCUSSION TITLE
Copy un-referenced files to _site?

### DISCUSSION THREAD

**Discussion by mikemahoney218** [quarto-dev/quarto-cli#1426] [Q&A] [answered]
On my previous distill website, files in the home directory (such as my resume, keybase.txt, and so on) were automatically copied into the _site directory. If I understand correctly, that's no longer the case, and files are only copied into _site if they're linked to by a qmd document. 

Is there a way for me to get these files to be copied into _site on render, without creating a "dummy" qmd page to link to the required files?

Here's a reprex for the issue -- notice how `required_file` is in `_site` on the distill branch:
https://github.com/mikemahoney218/example_site

But not on the quarto branch:
https://github.com/mikemahoney218/example_site/tree/quarto

Thanks!

**Comment by jjallaire** [comment, marked as answer]
Yes this does work differently from distill (which had the huge problem of potentially copying datasets and other assets that you don't want published). For Quarto there is a three-pronged approach:

1) Much better auto-detection of resources than we had in distill

2) Some well known files are automatically copied (`robots.txt`, `.nojekyll`, `CNAME`, `_redirects`)

3) If both of those fail you can manually specify `resources` to include in the `_site`: https://quarto.org/docs/websites/website-tools.html#site-resources

**Reply by mikemahoney218** [reply]
Fantastic, thank you! My google-fu failed me on this one.
